chronolog.py

```diff
@@ -0,0 +1,88 @@
+import os.path
+import regex as re
+
+import ChronologBackend.nlp as nlp
+import argparse
+from argparse import RawTextHelpFormatter
```

Comment:
Is it possible to just import what you need from argparse instead of importing all of it?